src/python/CMakeLists.txt

```diff
@@ -40,12 +41,28 @@ if(rpaths)
 endif()
 
 find_package(Python3 REQUIRED COMPONENTS Interpreter Development)
-install(FILES "${CMAKE_BINARY_DIR}/openvino_genai/__init__.py" "${CMAKE_BINARY_DIR}/openvino_genai/__version__.py" DESTINATION python/openvino_genai/ COMPONENT pygenai_${Python_VERSION_MAJOR}_${Python_VERSION_MINOR})
-install(TARGETS openvino_genai py_generate_pipeline LIBRARY DESTINATION python/openvino_genai/ COMPONENT pygenai_${Python_VERSION_MAJOR}_${Python_VERSION_MINOR})
+install(FILES "${OpenVINOGenAI_BINARY_DIR}/openvino_genai/__init__.py"
```

Comment:
`wheel.packages = ["src/python/openvino_genai"]` used to install \_\_init__.py into wheel. But how does it work now? :) init is installed although there's nothing doing that

Reply:
```
[tool.py-build-cmake.module]
directory = "src/python"
```